wallet/test/de/schildbach/wallet/util/viewModels/MainViewModelTest.kt

```diff
@@ -151,7 +156,7 @@ class MainViewModelTest {
         val viewModel = spyk(MainViewModel(
             mockk(), clipboardManagerMock, configMock, blockChainStateMock,
             exchangeRatesMock, walletDataMock, walletApp, appDatabaseMock, mockk(),
-            savedStateMock, mockk(), blockchainStateMock
+            savedStateMock, transactionMetadataMock, blockchainStateMock
```

Comment:
I think there was an error here, so this change was made.  will need to check again.